content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,536 +1,517 @@
 ---
 reviewers:
-- mikedanese
 - luxas
-- errordeveloper
-- jbeda
-title: Creating HA clusters with kubeadm
+- timothysc
+- detiber
+- chuckha
+title: Creating Highly Available Clusters with kubeadm
 content_template: templates/task
 ---
 
 {{% capture overview %}}
 
-This guide shows you how to install and set up a highly available Kubernetes cluster using kubeadm.
+This page explains two different approaches setting up a highly available Kubernetes
+cluster using kubeadm:
 
-This document shows you how to perform setup tasks that kubeadm doesn't perform: provision hardware; configure multiple systems; and load balancing.
+- With stacked masters. This approach requires less infrastructure. etcd members
+and control plane nodes are co-located.
+- With an external etcd cluster. This approach requires more infrastructure. The
+control plane nodes and etcd members are separated.
 
-{{< note >}}
-**Note:** This guide is only one potential solution, and there are many ways to configure a highly available cluster. If a better solution works for you, please use it. If you find a better solution that can be adopted by the community, feel free to contribute it back.
-{{< /note >}}
-
-{{% /capture %}}
+Your clusters must run Kubernetes version 1.11 or later.
 
-{{% capture prerequisites %}}
-
-- Three machines that meet [kubeadm's minimum requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for the masters
-- Three machines that meet [kubeadm's minimum requirements](/docs/setup/independent/install-kubeadm/#before-you-begin) for the workers
-- **Optional:** At least three machines that meet [kubeadm's minimum requirements](/docs/setup/independent/install-kubeadm/#before-you-begin)
-   if you intend to host etcd on dedicated nodes (see information below)
-- 1GB or more of RAM per machine (any less will leave little room for your apps)
-- Full network connectivity between all machines in the cluster (public or
-   private network is fine)
+{{< caution >}}
+**Caution**: This page does not address running your cluster on a cloud provider.
+In a cloud environment, neither approach documented here works with services of type 
+LoadBalancer, or with dynamic PersistentVolumes.
+{{< /caution >}}
 
```

Comment:
I think I've captured the original meaning more clearly here. The only thing I'm wondering: are these "services" K8s Service objects? Would it be even clearer to say so? I'm assuming a pretty sophisticated/advanced audience, that understands the differences between these things and the load balancer/s that you need to set up for your control plane/s. But I had to think it through before I understood. I am not, however, the intended audience.

Reply:
Services in this context are indeed Kubernetes Service Objects.